src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -90,6 +90,7 @@
       <Link>Common\System\RandomDataGenerator.cs</Link>
     </Compile>
     <Compile Condition="'$(TargetGroup)'=='netstandard1.7'" Include="Array.1.7.Tests.cs" />
+    <Compile Condition="'$(TargetGroup)'=='netstandard1.7'" Include="Enum.1.7.Tests.cs" />
```

Comment:
Can you break these into a separate ItemGroup and put the condition on the ItemGroup instead of the individual Compile items?